html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email"
+          >Email: <br />
+          <input type="text" id="email" />
+        </label>
+      </div>
+      <div>
+        <label for="why"
+          >Why is coding cool?<br />
+          <select>
+            <option>Select an option</option>
+            <option value="interesting">Because it's interesting!</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </label>
+      </div>
+      <div>
+        <div>Favorite Language (Choose one)</div>
+        <input type="radio" id="html" name="language" value="html" /><label
```

Comment:
here you want to either put a for attribute on the labels which match the id attribute, or wrap each of the radio inputs with a label. That is to make sure the click functionality of the label works